torch/_inductor/config.py

```diff
@@ -45,6 +45,22 @@
 # enable pattern match+replace optimizations
 pattern_matcher = True
 
+# register custom graph optimizatin pass hook. so far, pre/post passes are
+# only applied before/after pattern_matcher in post_grad_passes.
+#
+# def my_custom_pre_pass(graph: torch.fx.graph.Graph):
+#     # my custom graph optimization pass
+#     ...
+#
+# def my_custom_post_pass(graph: torch.fx.graph.Graph):
+#     # my custom graph optimization pass
+#     ...
+#
+# torch._inductor.config.custom_pre_pass = my_custom_pre_pass
+# torch._inductor.config.custom_post_pass = my_custom_post_pass
+custom_pre_pass = None
+custom_post_pass = None
```

Comment:
since they are for postgrad, rename them to `custom_post_grad_pre_pass` and `custom_post_prad_post_pass` to be more specific?

Reply:
I thought of it. What do you think of? `post_grad_custom_pre_pass` `post_grad_custom_post_pass`.